src/Views/EncryptView.vala

```diff
@@ -32,7 +32,10 @@ public class EncryptView : AbstractInstallerView {
     }
 
     construct {
-        var image = new Gtk.Image.from_icon_name ("drive-harddisk", Gtk.IconSize.DIALOG);
+        var image = new Gtk.Image.from_icon_name ("drive-harddisk", Gtk.IconSize.DIALOG) {
```

Comment:
Should be INVALID since it's a different size than the stock sizes

```suggestion
        var image = new Gtk.Image.from_icon_name ("drive-harddisk", INVALID) {
```

Reply:
Gotcha, I saw a couple other views use Dialog and override the size. Should I fix those as well and just rename this to icon clean-up?

Reply:
If you want to change the others too sure, but I'm not super worried about it since this property goes away in Gtk4 anyways

Reply:
Certain ones need the align and some don't so I'll leave it